README.md

```diff
@@ -79,6 +79,47 @@ Note there are various caveats to using this, which you can read about in detail
 - https://github.com/pavouk-0/postgres-xl-docker/issues/27
 - https://github.com/pavouk-0/postgres-xl-docker/pull/28
 
+## Clustering (Kubernetes; Automatically)
+
+So as ```docker-stack``` doesn't support ```depends_on``` option, ```docker-compose.stack.yml``` uses ```wait-for-it``` script. It delay main command until GTM node is ready.
+
+Also ```pgpool``` was added to ```docker-compose.stack.yml```. It sees to **coordinators nodes** and is making base enter point.
+
+```sh
+# TO UP
+docker stack deploy --orchestrator=kubernetes --namespace=default -c docker-compose.stack.yml pxl_stack
+# After you deployed pxl_stack, you need to initialize it (if it didn't do early or you purged PVC)
+./bin/init-eg-stack
```

Comment:
Typically, the `init-eg-*` scripts are just that: examples (e.g.), since of course a completely different topology might be required (this image supports fewer or more than 2 coordinators or datanodes).

Reply:
Yes of course, but all this solution is sample.
And to configure it for user's application all should change: `docker-compose`, `init-eg-*`.

I can add comment "for automatic configure it to test architecture, you can use it"